Code/Editor/Undo/Undo.cpp

```diff
@@ -307,7 +306,7 @@ void CUndoManager::Redo(int numSteps)
 
     if (m_bRecording || m_bSuperRecording)
```

Comment:
As a side note I think o3de is using another undo system so it might be possible to nuke this file

Reply:
Yeah just checked we still have a few occurences in LyShine and Trackview

Reply:
They forward their state to each other. There are still a couple of places where the old info system is used. I might of deleted them already.

Reply:
Maybe you can add a comment at the top of this class to state that it is deprecated and should use UndoStack from Framework\AzToolsFramework\AzToolsFramework\Undo\UndoSystem.h

Reply:
there is a rfc for deprecation notice i guess the pattern is to remove something in two releases. i guess something like this doesn't need to have a notice i'm not sure ...

https://github.com/pollend/o3de/commit/45bdd38c55e94a91989be733aa68722eadef699f#diff-176a2b2d54d4b07b48d218a442ef716a5dee590007aecb10439b6f904f847cd8

Reply:
I think its probably better to track this in a github issue?